src/Data/Nat/Apartness.agda

```diff
@@ -0,0 +1,59 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- An apartness relation for ℕ
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Nat.Apartness where
+
+open import Relation.Binary.Definitions using (Irreflexive; Symmetric)
+open import Data.Nat.Base using (ℕ; zero; suc)
+open import Relation.Binary.PropositionalEquality
+open import Data.Sum using (inj₁; inj₂; map)
+open import Data.Product using (_,_)
+open import Function using (_∘_)
+
+import Relation.Binary.Apartness as A
+open A.Structures
+
+
+data _#_ : ℕ → ℕ → Set where
```

Comment:
I'm a bit confused about why this definition is needed. Is it not just the negation of propositional equality?

Reply:
You are 100% correct --- I could easily prove that prop equality and the negation of this relation were isomorphic (that's what made it tight in fact...). I've removed this file.